modules/core/src/lib/loader-utils/auto-detect-loader.js

```diff
@@ -4,22 +4,45 @@ const EXT_PATTERN = /[^\.]+$/;
 // Search the loaders array argument for a loader that matches extension or text
 export function autoDetectLoader(url = '', text, loaders) {
   // Get extension
-  let extension = url.match(EXT_PATTERN);
-  if (extension && extension[0]) {
-    extension = extension[0].toLowerCase();
+  const match = url.match(EXT_PATTERN);
```

Comment:
Would be nice to support `http://example.com/file.glb?parameter=1`.
You can try
```
x = new URL('http://example.com/file.glb?load=1')
x.pathname
```